SPECS/xcp-ng-xapi-storage.spec

```diff
@@ -32,25 +32,29 @@ make
 cd build
 %make_install
 
-%post
-%systemd_post qemuback.service
-
-%preun
-%systemd_preun qemuback.service
-
-%postun
-%systemd_postun_with_restart qemuback.service
+#%post
+#%systemd_post qemuback.service
+#
+#%preun
+#%systemd_preun qemuback.service
+#
+#%postun
+#%systemd_postun_with_restart qemuback.service
 
 %files
 %license LICENSE README.md
-%{_bindir}/qemuback.py
+#%{_bindir}/qemuback.py
 %{_docdir}/xcp-ng-xapi-storage/
 %{_libexecdir}/xapi-storage-script/
 %{_prefix}/lib/python2.7/site-packages/xapi/storage/libs/
 %{_prefix}/lib/python2.7/site-packages/xcp_ng_xapi_storage_libs-*-py2.7.egg-info
-%{_prefix}/lib/systemd/system/qemuback.service
+#%{_prefix}/lib/systemd/system/qemuback.service
 
 %changelog
+* Thu Apr 04 2024 Yann Dirson <yann.dirson@vates.tech> - 1.2.0-1
```

Comment:
(about first commit) When we know we're not going to build from this commit, we usually make the comment look WIP, so that one could believe, by reading this commit, that it wa final.

Example:

```
* next - 1.2.0-1
- change 1
- change 2
- (optionally) TODO: change3
```

Reply:
fixed